lefthook.yml

```diff
@@ -1,50 +1,34 @@
-# cSpell:word commitlint typecheck stylelint
-pre-commit:
-  parallel: true
-  commands:
-    prettier:
-      glob: '*.{js,jsx}'
-      run: npm run prettier:ci {staged_files}
-    eslint:
-      glob: '*.{js,jsx}'
-      run: npm run eslint:ci {staged_files}
-    stylelint:
-      glob: './**/*.styled.jsx' # For CSS-in-JS
-      run: npm run stylelint:ci {staged_files}
-    spelling:
-      glob: '*.{js,jsx,md}'
-      run: npm run spelling:ci {staged_files}
-    markdown-link-check:
-      glob: '*.md'
-      run: npx markdown-link-check {staged_files}
+EXAMPLE USAGE
+Refer for explanation to following link:
+https://github.com/evilmartians/lefthook/blob/master/docs/full_guide.md
 
```

Comment:
Is this configuration giving you an error?

Reply:
No

Reply:
i just uncomment lefthook file and push the code again.

Reply:
Some things were deleted by mistake, you can copy the original content from dev branch and paste here and do the push again:

https://github.com/Azordev/maps-project/blob/dev/lefthook.yml

Reply:
Ok